### DISCUSSION TITLE
Cannot serialize non-str key

### DISCUSSION THREAD

**Discussion by baichishow** [aio-libs/aiohttp#5982] [Q&A] [answered]
third API header like this：
{'OK-ACCESS-SIGN': "b'ocJRoPZz5T1rohpNQYVjb/jDRslfwppRgDACs7O0mVA='", }
when using aiohttp , raise：‘ Cannot serialize non-str key b'ocJRoPZz5T1rohpNQYVjb/jDRslfwppRgDACs7O0mVA='

i can't just use str() to convert it , which will cause the api call error...

can anyone solve it?

**Comment by Dreamsorcerer** [comment, marked as answer]
If you want help, you'll have to produce some code which demonstrates the problem (and allows us to immediately run the code ourselves without guessing). Nobody here has time to figure out the problem from such a small snippet of information.
